yal/askaquestion.py

```diff
@@ -92,7 +92,7 @@ async def state_coming_soon(self):
         return EndState(
             self,
             self._("Coming soon..."),
-            next=self.START_STATE,
+            next="state_aaq_start",
```

Comment:
This is a nit-picking comment since this state shouldn't be reachable: won't this send them into an infinite loop? because they get here from `state_aaq_start` if the url isn't configured

Reply:
Ah true, I'll change this one back and merge, since it's a small change